### ISSUE TITLE
Fix Tests on EKS Brokerpak Branch and merge into main

### ISSUE THREAD

**Issue by nickumia-reisys** [GSA/data.gov#3706]
[We did a bunch of delicate Terraform surgery to get our static deployment of EKS working](https://github.com/GSA/datagov-brokerpak-eks/pull/82) and now we need to bring all that work back into the brokerpak workflow.

## How to reproduce

1. Take a look at [this PR](https://github.com/GSA/datagov-brokerpak-eks/pull/82).

## Expected behavior

Tests are passing and the PR was merged

## Actual behavior

Tests are failing and the PR continues to accrete commits, adding to its size and mass, like a snowball running away downhill, soon to crush a mountain village in tragedy.

## Sketch

[Notes or a checklist reflecting our understanding of the selected approach]

**Comment by mogul** [comment]
Currently blocked by https://github.com/18F/dns/pull/593
